crates/turbopack-core/src/issue/mod.rs

```diff
@@ -73,6 +73,42 @@ impl Display for IssueSeverity {
     }
 }
 
+#[derive(Clone, Debug)]
+#[turbo_tasks::value(shared)]
+pub enum StyledString {
+    Line(Vec<StyledString>),
+    String(String),
+    Pre(String),
+    Strong(String),
+}
+
+impl DeterministicHash for StyledString {
+    fn deterministic_hash<H: turbo_tasks_hash::DeterministicHasher>(&self, state: &mut H) {
+        match self {
+            StyledString::Line(parts) => {
+                for part in parts {
+                    part.deterministic_hash(state);
+                }
+            }
+            StyledString::String(s) | StyledString::Pre(s) | StyledString::Strong(s) => {
+                // Normalize syspaths from Windows. These appear in stack traces.
+                s.replace('\\', "/").deterministic_hash(state);
```

Comment:
I don't think this is the right spot for this, whatever created the `StyledString` should have already normalized.

Reply:
Technically `Issue`s should not have platform specific content. I think this is primarily caused by embedding node.js errors/stack traces into issues, which are platform dependent.

I would recommend to just remove this normalization here.